ledger-tool/src/main.rs

```diff
@@ -388,6 +393,78 @@ fn graph_forks(
     dot.join("\n")
 }
 
+fn analyze_column<T: solana_ledger::blocktree_db::Column>(
+    db: &Database,
+    name: &str,
+    key_size: usize,
+) -> Result<(), String> {
+    let mut key_hist = histogram::Histogram::new();
+    let mut key_tot: u64 = 0;
+    let mut val_hist = histogram::Histogram::new();
+    let mut val_tot: u64 = 0;
+    let mut row_hist = histogram::Histogram::new();
+    let a = key_size as u64;
+    for (_x, y) in db.iter::<T>(blocktree_db::IteratorMode::Start).unwrap() {
+        let b = y.len() as u64;
+        key_hist.increment(a).unwrap();
```

Comment:
Will the key size ever be different? Wondering how useful it would be to have this in the histogram.

Reply:
Ah, thank you for noticing that. I thought "maybe" before I discovered it's currently "no". I'll remove the histogram for `key`, it's easy enough to replace if we ever have variable key length.

Reply:
(key historgram removed)